src/modules/projectContext.js

```diff
@@ -1,82 +1,55 @@
-import { ProjectImageContext } from './projectImageContext.js';
-import { ProjectSearchContext } from './projectSearchContext.js';
-import { TraceContext } from './traceContext.js';
+import { AbstractProjectContext } from './abstractProjectContext.js';
+import { SessionManager } from './sessionManager.js';
 
-class ProjectContext {
+class ProjectContext extends AbstractProjectContext {
```

Comment:
I like the similar naming to hansken.py here as well.

Reply:
By accident :)